week_2/practice/src/components/TodoList.js

```diff
@@ -5,15 +5,25 @@ export default class TodoList extends Component {
   render() {
     return (
       <section className='todo-section-container'>
+        <button onClick={ this.props.handleCompleted }>Show Completed</button>
```

Comment:
You also did a great job here with the toggle button but I encourage you to update the button text based on the current `isCompleted` flag  just for a better UX
``` javascript
<button onClick={ this.props.handleCompleted}>
     {this.props.isCompleted ? 'Show All' : 'Show Completed' }
</button>
```

Reply:
Yes , this is better for ux
Thank you very much